device-types/HPE/ProLiant-DL385-Gen10-Plus-v2.yaml

```diff
@@ -0,0 +1,57 @@
+---
+manufacturer: HPE
+model: ProLiant DL385 Gen10 Plus v2
+slug: hpe-proliant-dl385-gen10-plus-v2
+u_height: 2
+is_full_depth: true
+airflow: front-to-rear
+front_image: true
+rear_image: true
+console-ports:
+  - name: VGA
+    type: other
```

Comment:
VGA is not a console port type. Please remove this

Reply:
fixed